.github/workflows/create-urgent-hotfix-img.yml

```
@@ -0,0 +1,301 @@
+    types: [ opened, synchronize, reopened, edited ]
+    branches:
+      - main
+  workflow_dispatch:
+    inputs:
+      branch:
+        description: 'The branch to create hotfix images from (ignored if commit is specified)'
+        required: false
+        type: string
+      commit:
+        description: 'The specific commit SHA to create hotfix images from (must be full 40-character SHA - takes precedence over branch)'
+        required: false
+        type: string
+      support-ticket-id:
+        description: 'The support ticket ID (e.g. SUPPORT-12345, JIRA-123, ticket-456)'
+        required: true
+        type: string
+      components_with_versions:
+        description: 'Components to build with versions (e.g. camunda:8.6.1,operate:8.6.1)'
+        required: true
+        type: string
+
+defaults:
+  run:
+    shell: bash
+
+env:
+  HOTFIX_REGISTRY: "registry.camunda.cloud/hotfixes"
+  PLATFORMS: "linux/amd64,linux/arm64"
+
+jobs:
+  validate-inputs:
+    name: Validate Inputs
+    runs-on: ubuntu-latest
+    outputs:
+      branch: ${{ steps.parse.outputs.branch }}
+      commit: ${{ steps.parse.outputs.commit }}
+      support-ticket-id: ${{ steps.parse.outputs.support-ticket-id }}
+      components-with-versions: ${{ steps.parse.outputs.components-with-versions }}
+      components-matrix: ${{ steps.parse.outputs.matrix }}
+    steps:
+      - name: Validate and parse components
+        id: parse
+        run: |
+          BRANCH="${{ github.event.inputs.branch }}"
+          COMMIT="${{ github.event.inputs.commit || 'f7ff978967f6ccd4143b9b0450a5f0aa036dd4ba' }}"
+          TICKET_ID="${{ github.event.inputs.support-ticket-id || 'pr-test' }}"
+          COMPONENTS="${{ github.event.inputs.components_with_versions || 'tasklist:8.8.0-TEST,optimize:8.8.0-TEST,camunda:8.8.0-TEST,zeebe:8.8.0-TEST,operate:8.8.0-TEST' }}"
+
+          [[ -z "$BRANCH" && -z "$COMMIT" ]] && { echo "❌ Either branch or commit required"; exit 1; }
+          [[ -n "$COMMIT" && ! "$COMMIT" =~ ^[a-f0-9]{40}$ ]] && { echo "❌ Commit must be a full 40-character SHA (got: $COMMIT)"; exit 1; }
+          [[ -z "$TICKET_ID" ]] && { echo "❌ Support ticket ID required"; exit 1; }
+          [[ ! "$TICKET_ID" =~ ^[a-zA-Z0-9_-]+$ ]] && { echo "❌ Invalid ticket ID format"; exit 1; }
+          [[ -z "$COMPONENTS" ]] && { echo "❌ Components required"; exit 1; }
+
+          {
+            echo "branch=$BRANCH"
+            echo "commit=$COMMIT"
+            echo "support-ticket-id=$TICKET_ID"
+            echo "components-with-versions=$COMPONENTS"
+          } >> "$GITHUB_OUTPUT"
+
+          VALID_COMPONENTS=()
+          IFS=',' read -ra ARRAY <<< "$COMPONENTS"
+          for ITEM in "${ARRAY[@]}"; do
+            ITEM=$(echo "$ITEM" | xargs)
+            if [[ "$ITEM" =~ ^([^:]+):(.+)$ ]]; then
+              COMP="${BASH_REMATCH[1]}"
+              case "$COMP" in
+                zeebe|camunda|optimize|operate|tasklist) VALID_COMPONENTS+=("$ITEM") ;;
```

```suggestion
                camunda|optimize|operate|tasklist|zeebe)
```
(alphabetical order for readability)